uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/zones/OrchestratorZoneControllerMockMvcTests.java

```diff
@@ -187,9 +189,11 @@ void testGetZone() throws Exception {
 
         ConnectionDetails expectedConnectionDetails = new ConnectionDetails();
         expectedConnectionDetails.setSubdomain(SUB_DOMAIN_NAME);
-        expectedConnectionDetails.setUri("http://" + SUB_DOMAIN_NAME + ".localhost");
-        expectedConnectionDetails.setDashboardUri("http://localhost:8080/dashboard");
-        expectedConnectionDetails.setIssuerId(expectedConnectionDetails.getUri() + "/oauth/token");
+        String uri = "http://" + SUB_DOMAIN_NAME + ".localhost:8080/uaa";
+        String issuerId = "http://" + SUB_DOMAIN_NAME + ".localhost:8080/uaa";
```

Comment:
For mock testing, uaa is configured using this [uaa.yml](https://github.com/GESoftware-CF/uaa/blob/predix_extensions_75.18.1/uaa/src/test/resources/test/config/uaa.yml). Is it possible to make uri and issuerId different? Otherwise, it's not really tested that in the actual connection details they are constructed independently.

Reply:
Thanks! it is not working with different uri and issuerId.

Its enough to test with unit tests only i think. Hence removing these checks from mock mvc tests and integration tests.

Reply:
MockMvc tests are configured with integration_test_properties.yml, I updated issuer id in this file but other tests are failing due to this, so I reverted back.